README.md

```diff
@@ -15,11 +15,13 @@ using a key stored with the local SSH Agent (using an [`SSHAgentSigner`][6].
 To construct a Signer, use the `New*` range of methods in the `authentication`
 package. In the case of `authentication.NewSSHAgentSigner`, the parameters are
 the fingerprint of the key with which to sign, and the account name (normally
-stored in the `SDC_ACCOUNT` environment variable). For example:
+stored in the `SDC_ACCOUNT` environment variable). There is also support for 
```

Comment:
Alright, when do we want to have our watershed moment and move from `SDC_ACCOUNT` in docs to `TRITON_ACCOUNT` in docs?  This isn't an issue with this PR, however.  Opening joyent/triton-go#83 to track this issue.

Reply:
I will follow up with a PR to solve this

Reply:
We may as well do this now too.

Reply:
I would advocate for that before we tag `0.4.0`, yes.